control-plane/api-gateway/gatekeeper/service.go

```diff
@@ -32,12 +30,12 @@ func (g *Gatekeeper) upsertService(ctx context.Context, gateway gwv1beta1.Gatewa
 		return g.deleteService(ctx, types.NamespacedName{Namespace: gateway.Namespace, Name: gateway.Name})
 	}
 
-	service := g.service(gateway, gcc)
+	desiredService := g.service(gateway, gcc)
```

Comment:
These were renamed to match the [naming for V2](https://github.com/hashicorp/consul-k8s/pull/3537/files#diff-89e20595aea84d3419e94fea412b6d12ce1a1d1e3a070470f87c519a6ee3b876R81), as well as to make more understandable